pyneuroml/utils/__init__.py

```diff
@@ -473,6 +473,61 @@ def rotate_cell(
 
     return newcell
 
+def translate_cell_to_origin(
+    cell: neuroml.Cell,
+    inplace: bool = False,
+) -> neuroml.Cell:
+    """Return a new cell object translated so that the some is at the origin.
+
+    :param cell: cell object to translate
```

Comment:
Let's add an `versionadded` info bit here too, so users know what version this is available from.

I think it'll be:

``` rst
.. versionadded:: 1.2.13

```

(search the sources for `versionadded` just to be sure)

Reply:
Hm, i'm not familiar with this, could you please elaborate a bit what I need to look for?

Reply:
Take a look at this: https://github.com/NeuroML/pyNeuroML/blob/bfb0197943efbff74e6dc1fee3cd102e6d2433d7/pyneuroml/runners.py#L1264

and the generated doc:

https://pyneuroml.readthedocs.io/en/development/pyneuroml.pynml.html#pyneuroml.pynml.generate_sim_scripts_in_folder

Sphinx docs: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded

Reply:
Ok, got it, so since currently we are on v1.2.12, I should the next one which will be v1.2.13, right?

Reply:
Yes. We bump the release in setup.cfg to the next version after making a release. So that's at 1.2.13 now.